src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/cache.max-stale/accept/client.rpt

```diff
@@ -0,0 +1,73 @@
+#
```

Comment:
nit, consider renaming this script cache.max-stale.with.value to be inline with the without value script